tests/sentry/middleware/integrations/parsers/test_bitbucket.py

```diff
@@ -25,9 +25,6 @@ def get_response(self, req: HttpRequest) -> HttpResponse:
     region = Region("us", 1, "https://us.testserver", RegionCategory.MULTI_TENANT)
     region_config = (region,)
 
-    def setUp(self):
-        super().setUp()
```

Comment:
Nit: how will this thing then, setup?

Reply:
Wouldn't the setup from the parent be called?

Reply:
i'm just being checky